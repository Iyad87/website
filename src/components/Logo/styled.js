import styled from 'styled-components';
import GatsbyLink from 'gatsby-link';

export const YLD = styled.h1`
  color: #111;
  font-weight: bold;
  margin: 0;
`;

export const Link = styled(GatsbyLink)`
  text-decoration: none;

  &:visited {
    color: inherit;
  }
`;
