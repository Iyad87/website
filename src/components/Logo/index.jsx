import React from 'react';
import { YLD, Link } from './styled';

export default function Logo() {
  return (
    <Link to="/">
      <YLD>yld</YLD>
    </Link>
  );
}
