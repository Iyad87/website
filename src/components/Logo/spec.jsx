import React from 'react';
import { shallow } from 'enzyme';
import Logo from './index';

test('components/Logo snapshot', () => {
  const wrapper = shallow(<Logo />);

  expect(wrapper).toMatchSnapshot();
});
