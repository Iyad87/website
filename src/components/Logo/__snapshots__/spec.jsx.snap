// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Logo snapshot 1`] = `
<Styled(GatsbyLink)
  to="/"
>
  <styled.h1>
    yld
  </styled.h1>
</Styled(GatsbyLink)>
`;
